hazelcast/src/test/java/com/hazelcast/journal/AbstractEventJournalBasicTest.java

```diff
@@ -221,13 +218,7 @@ public void receiveLoadedEventsWhenLoadAll() throws Exception {
         testLoadAll(LOADED);
     }
 
-    @Test
-    public void receiveAddedEventsWhenLoadAll() throws Exception {
-        testLoadAll(ADDED);
-    }
-
     private void testLoadAll(EventType expectedEventType) throws Exception {
```

Comment:
Think this method is not needed anymore and it's content can be moved into `receiveLoadedEventsWhenLoadAll()`.

Reply:
done.